### PR TITLE
Update locale-tr.js

### PR DESCRIPTION
**Pull request by bitsacer**
Asset settlement

### CODE REVIEW

File: web/app/assets/locales/locale-tr.js [hunk 1]
```diff
@@ -32,7 +32,7 @@
         pending: "sırada %(blocks)s blok var"
     },
     propose: "teklif et",
-    cancel: "iptal et",
```
Comment (svk31) [inline web/app/assets/locales/locale-tr.js:35]: You have to keep this comma, if not the file is not valid JSON.

Comment (bitsacer) [inline web/app/assets/locales/locale-tr.js:35]: Hey, does that every time  happen with the same comma ? that is what I download from transifex after making changes. 

Have a great day !

Mithat Arslan
mithars@gmail.com

Comment (bitsacer) [inline web/app/assets/locales/locale-tr.js:35]: Apparently it is , sorry.

Have a great day !

Mithat Arslan
mithars@gmail.com

Comment (svk31) [inline web/app/assets/locales/locale-tr.js:35]: There was one error like this in an earlier PR, not sure if it was in the same place

**Comment by bitsacer** [comment]
Can we expect the translation to be updated today along with the web gui ? 

Thanks

Have a great day !

Mithat Arslan
mithars@gmail.com

**Comment by svk31** [comment]
Yes

**Comment by bitsacer** [comment]
Great, thanks.

Have a great day !

Mithat Arslan
mithars@gmail.com

**Pull request merged**